src/lib/app-switcher/app-switcher-menu/app-switcher-menu.component.scss

```diff
@@ -29,3 +29,13 @@
     }
   }
 }
+
+// Swap logo version based on app theme
+.dark-theme :host ::ng-deep .logo-icon svg {
```

Comment:
We do not want to be doing this every time we consume that logo.

Reply:
Could move this up into overrides and not need the :host ::ng-deep if we plan to have this logo in other places.